smac_planner/src/a_star.cpp

```diff
@@ -14,6 +15,10 @@
 
 #include <omp.h>
 
+#include <ompl/base/ScopedState.h>
```

Comment:
These should be in quotes, linting thing that cpplint will complain about

Reply:
`ament_uncrustify` is happy with this when I run it..?

Reply:
Sorry, I meant `ament_cpplint` is happy with this (so is `ament_uncrustify`)